options/options.js

```diff
@@ -144,3 +160,53 @@ function updateState() {
     option.value = newValue;
   });
 }
+
+function moveUp(id) {
+  // move the element up
+  var id_temp = (id.toString()).replace("-move-up","");
+  document.getElementById(id_temp+"-priority").value = parseInt(document.getElementById(id_temp+"-priority").value)-1;
+  // call "onchange" event (forced)
+  if ("createEvent" in document) {
+    var evt = document.createEvent("HTMLEvents");
+    evt.initEvent("change", false, true);
+    document.getElementById(id_temp+"-priority").dispatchEvent(evt);
+  }
+  else
+    document.getElementById(id_temp+"-priority").fireEvent("onchange");
+  setFocusRowTemp(parseInt(document.getElementById(id_temp+"-priority").value)-1);
+  document.getElementById(id_temp+"-priority").focus();
+}
+
+function moveDown(id) {
+  // move the element down
+  var id_temp = (id.toString()).replace("-move-down","");
+  document.getElementById(id_temp+"-priority").value = parseInt(document.getElementById(id_temp+"-priority").value)+1;
```

Comment:
this code is a duplicate with the above method better create another function to simplify the maintenance.

Reply:
Also associate `document.getElementById(id_temp+"-priority")` to a var for performance and readibility